src/common/exception/src/exception_code.rs

```diff
@@ -24,7 +24,7 @@ pub static ABORT_SESSION: u16 = 1042;
 pub static ABORT_QUERY: u16 = 1043;
 
 macro_rules! build_exceptions {
-    ($($body:ident($code:expr)),*$(,)*) => {
+    ($($(#[$meta:meta])* $body:ident($code:expr)),*$(,)*) => {
```

Comment:
Hmm? what's this meta for? I did not find its usage.

Reply:
`///` will be converted to `#[doc]`, this meta helps capture them.

I will use them in the followimg pr.

Reply:
`Σ(⊙▽⊙"a`